src/esp/gfx/MaterialData.h

```diff
@@ -16,11 +16,19 @@ namespace gfx {
 
 struct MaterialData {};
 
-struct PhongMaterialData : public MaterialData {
+// This struct contains a subset of phong material fields (with the rest in
+// PhongMaterialData). We expose this struct to python so that scripts can
+// modify these fields at runtime.
+struct PhongMaterialInfo {
```

Comment:
I named this `PhongMaterialInfo` to be consistent with `LightInfo` which we also expose to python.